pkg/controller/v1beta1/inferenceservice/reconcilers/ingress/kube_ingress_reconciler.go

```diff
@@ -80,9 +80,15 @@ func createRawURL(client client.Client, isvc *v1beta1.InferenceService, authEnab
 			return nil, err
 		}
 	} else {
+		var scheme string
+		if authEnabled {
```

Comment:
not sure about it, why are you enforcing auth only with SSL?

Reply:
doing this, the code looks cleaner.

```go
var scheme string
		url = &apis.URL{
			Host:   getRawServiceHost(isvc, client),
			Scheme: "http",
			Scheme: scheme,
			Path:   "",
		}
		if authEnabled {
			url.Scheme = "https"
		} 
```

Reply:
certs : https://github.com/opendatahub-io/kserve/blob/master/pkg/controller/v1beta1/inferenceservice/reconcilers/service/service_reconciler.go#L176 

All reported ISVC status schemes are according to what is expected in each case according to https://github.com/opendatahub-io/kserve/pull/419#issuecomment-2524125126

Reply:
also, code change is included in latest commit